spec/facades/get_art_facade_spec.rb

```diff
@@ -1,29 +1,114 @@
+              {:mid=>"/m/0jjw", :description=>"Art", :score=>0.82273906, :topicality=>0.82273906},
+              {:mid=>"/m/083vt", :description=>"Wood", :score=>0.8041993, :topicality=>0.8041993},
+              {:mid=>"/m/0j62f", :description=>"Rectangle", :score=>0.7840926, :topicality=>0.7840926},
+              {:mid=>"/m/05y4t", :description=>"Paint", :score=>0.77848625, :topicality=>0.77848625}]}]},
+       "4d8b93394eb68a1b2c0010fa"=>
+        {:responses=>
+          [{:labelAnnotations=>
+             [{:mid=>"/m/07j7r", :description=>"Tree", :score=>0.8316971, :topicality=>0.8316971},
+              {:mid=>"/m/0jjw", :description=>"Art", :score=>0.82273906, :topicality=>0.82273906},
+              {:mid=>"/m/083vt", :description=>"Wood", :score=>0.8041993, :topicality=>0.8041993},
+              {:mid=>"/m/0j62f", :description=>"Rectangle", :score=>0.7840926, :topicality=>0.7840926},
+              {:mid=>"/m/05y4t", :description=>"Paint", :score=>0.77848625, :topicality=>0.77848625}]}]},
+       "4d8b937c4eb68a1b2c001722"=>
+        {:responses=>
+          [{:labelAnnotations=>
+             [{:mid=>"/m/07j7r", :description=>"Tree", :score=>0.8316971, :topicality=>0.8316971},
+              {:mid=>"/m/0jjw", :description=>"Art", :score=>0.82273906, :topicality=>0.82273906},
+              {:mid=>"/m/083vt", :description=>"Wood", :score=>0.8041993, :topicality=>0.8041993},
+              {:mid=>"/m/0j62f", :description=>"Rectangle", :score=>0.7840926, :topicality=>0.7840926},
+              {:mid=>"/m/05y4t", :description=>"Paint", :score=>0.77848625, :topicality=>0.77848625}]}]},
+       "4d8b93b04eb68a1b2c001b9d"=>
+        {:responses=>
+          [{:labelAnnotations=>
+             [{:mid=>"/m/07j7r", :description=>"Tree", :score=>0.8316971, :topicality=>0.8316971},
+              {:mid=>"/m/0jjw", :description=>"Art", :score=>0.82273906, :topicality=>0.82273906},
+              {:mid=>"/m/083vt", :description=>"Wood", :score=>0.8041993, :topicality=>0.8041993},
+              {:mid=>"/m/0j62f", :description=>"Rectangle", :score=>0.7840926, :topicality=>0.7840926},
+              {:mid=>"/m/05y4t", :description=>"Paint", :score=>0.77848625, :topicality=>0.77848625}]}]}}
+
+        expect(selection.class).to eq(Hash)
+        expect(selection).to eq(expected)
+    end
+  end 
+
+  context 'Find image Colors' do
+    it 'converts jpeg to colors' do
+      number = 5
+      size = 'large'
+
+      stub_1 = WebmockStubs.mock_art
+
+      stub_request(:get, "https://api.artsy.net/api/artworks?size=5").to_return(status: 200, body: stub_1, headers: {})
+      stub_request(:post, "https://api.artsy.net/api/tokens/xapp_token").to_return(status: 200, body: stub_1, headers: {})  
+      
+      stub_2 = WebmockStubs.mock_colors
+
+      stub_request(:post, "https://vision.googleapis.com/v1/images:annotate?key=#{ENV['key']}").to_return(status: 200, body: stub_2)
```

Comment:
Okay sweet. Once we merge this in I'll delete the old file that has our API key published.